docs/os/windows/windows-tools.md

```diff
@@ -1,5 +1,4 @@
-:warning: *Some of the information below is outdated ; please read [this forum wiki post](https://forum.qubes-os.org/t/wip-windows-qwt-user-reports/9687) for up to date instructions and users feedback.
-The content below will eventually be updated to reflect the newer 4.1 release and QWT recent development (contributors welcome!)*
+:warning: *The content below has been updated to reflect the newer 4.1 release and QWT recent development - see* [Installing and Using Windows Tools in Qubes R4.1](https://github.com/Qubes-Community/Contents/blob/master/docs/os/windows/windows-tools41.md)
```

Comment:
(first, thank you for your work).
I'm not sure I understand properly - you've created windows-{migrate,tools,vm}41.md files but the change above says that the old file(s)' content is updated to 4.1, yet there's a link to the newer 41 file. Shouldn't the warning's text be something like "Please see [this updated document](https://github.com/Qubes-Community/Contents/blob/master/docs/os/windows/windows-tools41.md) for instructions for Qubes 4.1" ?